.github/workflows/tailwind-native-host-evidence.yml

```
@@ -0,0 +1,219 @@
+          - rid: linux-arm64
+            binary_name: tailwindcss-linux-arm64
+          - rid: osx-x64
+            binary_name: tailwindcss-macos-x64
+          - rid: osx-arm64
+            binary_name: tailwindcss-macos-arm64
+          - rid: win-x64
+            binary_name: tailwindcss-windows-x64.exe
+    env:
+      PACKAGE_VERSION: ${{ inputs.package_version || format('0.0.0-native-host.{0}', github.run_id) }}
+      SOURCE_COMMIT: ${{ inputs.source_commit || github.sha }}
+      PROOF_DIRECTORY: ${{ runner.temp }}/tailwind-native-host-proof
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)